tex/gregoriotex-main.tex

```diff
@@ -1703,7 +1633,7 @@
 % Macros for compiling short snippets of GABC directly expressed in TeX
 
 \long\def\gre@gabcsnippet@option[#1]#2{%%% DEPRECATED by 4.1
-  \gre@deprecated{\protect\gabcsnippet  with initial-style argument}{\protect\gresetinitiallines}%
+  \gre@obsolete{\protect\gabcsnippet  with initial-style argument}{\protect\gresetinitiallines}%
```

Comment:
This is also not obsolete yet.  Deprecation of initial-style was introduced in develop.